src/Microsoft.AspNetCore.Routing/Matchers/DfaMatcherBuilder.cs

```diff
@@ -209,46 +218,56 @@ public override Matcher Build()
             var root = BuildDfaTree();
 
             var states = new List<DfaState>();
-            var tables = new List<JumpTableBuilder>();
-            AddNode(root, states, tables);
+            var tableBuilders = new List<(JumpTableBuilder pathBuilder, PolicyJumpTableBuilder policyBuilder)>();
```

Comment:
Ok I'm sure the diff is kinda confusing at this point if you're reading in order. We have two kinds of jump tables now:
- path jump tables
- policy jump tables

I want to make sure theres a lookup from state -> jump table, so I just make each state have a tuple of builders, because that seemed easiest.